src/Raven.Server/Documents/Sharding/ShardedDatabaseContext.cs

```diff
@@ -92,6 +105,34 @@ public void UpdateDatabaseRecord(RawDatabaseRecord record, long index)
             RachisLogIndexNotifications.NotifyListenersAbout(index, e: null);
         }
 
+        private bool AreElementsEqual(List<string> list1, List<string> list2)
+        {
+            if (list1.Count != list2.Count)
+                return false;
+
+            foreach (var item in list1)
+            {
+                if (list2.Contains(item) == false)
+                    return false;
```

Comment:
better to sort them and do `SequenceEqual` but N here is low so I guess it is acceptable..